bedrock/firefox/views.py

```diff
@@ -541,6 +542,17 @@ def get_template_names(self):
         return [template]
 
 
+def fxos(request):
+    locale = l10n_utils.get_locale(request)
+    old_home = 'firefox/os/index.html'
+    new_home = 'firefox/os/index-new.html'
+
+    if lang_file_is_active(new_home, locale):
```

Comment:
Ok so:

    lang_file = 'firefox/os/index-new'
    old_home = 'firefox/os/index.html'
    new_home = 'firefox/os/index-new.html'

    if lang_file_is_active(lang_file, locale):

is what we are after? @pmclanahan @flodolo